frontend/src/metabase/core/components/AccordionList/AccordionListCell.jsx

```diff
@@ -19,36 +20,39 @@ import {
   ListCellItem,
 } from "./AccordionListCell.styled";
 
-export const AccordionListCell = ({
-  style,
-  sections,
-  row,
-  onChange,
-  itemIsSelected,
-  itemIsClickable,
-  sectionIsExpanded,
-  canToggleSections,
-  alwaysExpanded,
-  toggleSection,
-  renderSectionIcon,
-  renderItemLabel,
-  renderItemName,
-  renderItemDescription,
-  renderItemIcon,
-  renderItemExtra,
-  renderItemWrapper,
-  showSpinner,
-  searchText,
-  onChangeSearchText,
-  searchPlaceholder = t`Find...`,
-  showItemArrows,
-  itemTestId,
-  getItemClassName,
-  getItemStyles,
-  searchInputProps,
-  hasCursor,
-  withBorders,
-}) => {
+export const AccordionListCell = forwardRef(function AccordionListCell(
```

Comment:
CellMeasurer now expects a component with `ref` as its children, so we have a single case of the `CellMeasurer` with the `AccordionListCell` as a children, i wrapped it within forwardRef